### DISCUSSION TITLE
用Qwen2.5进行lora的数据是什么呀

### DISCUSSION THREAD

**Discussion by acmilaner** [shibing624/pycorrector#532] [Q&A] [answered]
请问用Qwen2.5进行lora的训练数据是什么样的呀，直接用Qwen和lora后的，指标差异大吗？

**Comment by shibing624** [comment, marked as answer]
modelcard有写，是https://huggingface.co/datasets/shibing624/chinese_text_correction

**Reply by acmilaner** [reply]
谢谢，顺便请教下，看评测指标，基于Qwen进行lora的方法，平均的指标很好，但是在 SIGHAN-2015 上表现要差挺多，是什么原因呀，这种能通过修改lora数据集，来改善效果吗？ 


<br class="Apple-interchange-newline">

**Reply by shibing624** [reply]
可以在专门数据集上训练。测试集错误分布不同，sighan是台湾人出的数据集。

**Reply by acmilaner** [reply]
明白，谢谢～
